src/constants/affiliates.ts

```diff
@@ -1,3 +1,8 @@
-export const AFFILIATES_EARN_PER_MONTH = 1500;
+export const DEFAULT_AFFILIATES_EARN_PER_MONTH = 1500;
```

Comment:
nit: constants like these should also specify the unit they're in eg `DEFAULT_AFFILIATES_EARN_PER_MONTH_USD` (if thats the case)